src/components/govukComponents/Checkboxes.js

```diff
@@ -31,23 +31,26 @@ function Checkboxes(props) {
   }
 
   const innerHtml = props.items.map((checkbox, index) =>
-    <><div key={props.name + index} className="govuk-checkboxes__item">
-      <input
-        id={checkbox.conditional ? props.name + '-conditional-' + (index + 1) : props.name + (index + 1)}
-        className="govuk-checkboxes__input"
-        type="checkbox" name={props.name}
-        value={checkbox.value}
-        aria-controls={checkbox.conditional ? `conditional-${props.name}-conditional-${index + 1}` : ''}
-      />
-      <Label text={checkbox.text} classes="govuk-checkboxes__label" for={props.name + (index + 1)} />
-      {checkbox.hint ? <Hint classes="govuk-checkboxes__hint" {...checkbox.hint} /> : ''}
-    </div>
+    <React.Fragment key={props.name + index}>
```

Comment:
Big diff here, but all that's happened is moving the key to the fragment instead of the child div